### ISSUE TITLE
[Site Feature Request] RSS feed

### ISSUE THREAD

**Issue by finuex** [samanthaming/samanthaming.com#18]
### Feature Description

RSS feed

### Additional context

tolong tambahkan RSS feed pada web agar dapat saya pantau dari aplikasi seperti [feedly](https://www.feedly.com)

**Comment by samanthaming** [comment]
@finuex YES! I love this suggestion! It's on my TODO list. For now, I'v been posting updates on my social media page (Instagram and Twitter)...but having a RSS feed will be the direction I want to head into! I'm currently working on a few other things (new course 🤩), so once I have more free time, this is coming soon. Thanks for this request 👏

**Comment by jsbalsera** [comment]
Hi! I discovered your site just today and it's really awesome :-) Came to Github to propose RSS as a new feature, but decided to check first if someone else proposed it, so wanted to give a +1. Thank you!

**Comment by holtwick** [comment]
Your site and tidbits are awesome! Congratulations. An RSS/Atom feed would be great. I mostly consume blogs using https://reederapp.com/ and would love to add it. Thanks a lot for your great content!

**Comment by mharleydev** [comment]
+1 also

😬

**Comment by ShangHungTsai** [comment]
+1 also
